pom.xml

```diff
@@ -234,6 +234,11 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-logging-json</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkiverse.kerberos</groupId>
+      <artifactId>quarkus-kerberos</artifactId>
+      <version>2.2.0</version>
+    </dependency>
```

Comment:
This is what fixed `{"refId":6,"exceptionType":"sun.security.jgss.GSSExceptionImpl","message":"Unsupported mechanism requested: 1.2.840.113554.1.2.2"}`